src/lib/translator.ts

```diff
@@ -24,12 +24,10 @@ export class Translator implements TranslatorInterface {
     private _registry: PluginRegistry = {};
 
     private _locale: Intl.Locale;
-    private _pluralRule: Intl.PluralRules;
```

Comment:
the plural rule is intended to be "cached" to not create a new instance every time. This may be better to add it once for the fallback too instead of create all on the fly